svd-encoder/src/access.rs

```diff
@@ -0,0 +1,11 @@
+use super::{Element, Encode, EncodeError, XMLNode};
+
+impl Encode for crate::svd::Access {
+    type Error = EncodeError;
+
+    fn encode(&self) -> Result<Element, EncodeError> {
+        let mut elem = Element::new("access");
```

Comment:
I think it makes sense to put most string literals in statics,

```rust
static ELEMENT_ACCESS: &str = "access";
```

maybe those should live in svd-rs

Reply:
What's the point of this?

Reply:
to avoid sprinkling literals in multiple places, meaning eventual typos can be fixed in one place or even avoided. I guess the most obvious literal would be derivedFrom

this would mostly benefit users of svd-rs, but wouldnt bring any other advantages

Reply:
I don't really mind either way, but instead of statics perhaps it could be an enum that impls Into<&'static str> or something?

Reply:
This can be done separately